<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtendedControls</class>
 <widget class="QFrame" name="ExtendedControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Extended controls</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Qt::WindowMinimizeButtonHint; This is bad*/
background-color: #3c4249;
color: #f0f0f0;
font-family: &quot;Roboto&quot;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>0</y>
     <width>620</width>
     <height>600</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="LocationPage" name="location_page"/>
   <widget class="CellularPage" name="cellular_page"/>
   <widget class="BatteryPage" name="batteryPage"/>
   <widget class="TelephonyPage" name="telephonyPage"/>
   <widget class="VirtualSensorsPage" name="virtualSensorsPage"/>
   <widget class="DPadPage" name="dpadPage"/>
   <widget class="MicrophonePage" name="microphonePage"/>
   <widget class="FingerPage" name="finger_page"/>
   <widget class="RecordScreenPage" name="recordScreenPage"/>
   <widget class="SettingsPage" name="settingsPage"/>
   <widget class="HelpPage" name="helpPage"/>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="locationGroup_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="title">
      <string>Hidden page that holds future widgets for other pages</string>
     </property>
     <property name="ColorGroup" stdset="0">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="locationGroup_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="title">
      <string>Cellular page:</string>
     </property>
     <property name="ColorGroup" stdset="0">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="cell_delayGroupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>122</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="title">
      <string>Network delay (msec)</string>
     </property>
     <property name="ColorGroup" stdset="0">
      <string notr="true">Title</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cell_delayBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>280</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px 3px 3px 3px;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="report_metrics" stdset="0">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>35-200 (UMTS/3G)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>150-500 (GPRS)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>80-400 (EDGE/EGPRS)</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="themeTitle_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>262</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="title">
      <string>On-screen keyboard</string>
     </property>
     <property name="ColorGroup" stdset="0">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="set_keyboardBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>280</y>
       <width>211</width>
       <height>24</height>
      </rect>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="ColorGroup" stdset="0">
      <string>Toggle</string>
     </property>
     <property name="report_metrics" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="locationGroup_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;</string>
     </property>
     <property name="title">
      <string>Settings page:</string>
     </property>
     <property name="ColorGroup" stdset="0">
      <string>Title</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="telephoneButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>138</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Phone</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/phone</normaloff>:/light/phone</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>phone</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="batteryButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>96</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Battery</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/battery</normaloff>:/light/battery</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>battery</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="locationButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>12</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Location</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/location</normaloff>:/light/location</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>location</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cellularButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>54</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Cellular</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/signal_cellular_3_bar</normaloff>:/light/signal_cellular_3_bar</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>signal_cellular_3_bar</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fingerButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>264</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Fingerprint</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/fingerprint</normaloff>:/light/fingerprint</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>fingerprint</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dpadButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Directional pad</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/dpad</normaloff>:/light/dpad</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>dpad</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
  </widget>
  <widget class="Line" name="VertDivider">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>0</y>
     <width>4</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : #808080</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="ColorGroup" stdset="0">
    <string notr="true">GradientDivider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>396</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Settings</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/settings</normaloff>:/light/settings</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>settings</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spacer2Button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>194</width>
     <height>250</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTabTitle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Help</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/help</normaloff>:/light/help</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>help</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="spacer1Button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>194</width>
     <height>12</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTabTitle</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>348</y>
     <width>194</width>
     <height>4</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>HorizDivider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="virtSensorsButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>306</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Virtual sensors</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/virtSensors</normaloff>:/light/virtSensors</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>virtSensors</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="microphoneButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>222</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Microphone</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/mic</normaloff>:/light/mic</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>mic</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recordScreenButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>352</y>
     <width>194</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left; padding: 0px 0px 0px 16px;</string>
   </property>
   <property name="text">
    <string>  Record Screen</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/light/record_screen</normaloff>:/light/record_screen</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="themeIconName" stdset="0">
    <string>record_screen</string>
   </property>
   <property name="ColorGroup" stdset="0">
    <string>MajorTab</string>
   </property>
   <property name="report_metrics" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>spacer2Button</zorder>
  <zorder>spacer1Button</zorder>
  <zorder>recordScreenButton</zorder>
  <zorder>settingsButton</zorder>
  <zorder>helpButton</zorder>
  <zorder>fingerButton</zorder>
  <zorder>dpadButton</zorder>
  <zorder>microphoneButton</zorder>
  <zorder>telephoneButton</zorder>
  <zorder>batteryButton</zorder>
  <zorder>cellularButton</zorder>
  <zorder>locationButton</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>VertDivider</zorder>
  <zorder>line</zorder>
  <zorder>virtSensorsButton</zorder>
  <zorder>microphoneButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CellularPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/cellular-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BatteryPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/battery-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TelephonyPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/telephony-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FingerPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/finger-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HelpPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/help-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DPadPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/dpad-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MicrophonePage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/microphone-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/settings-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LocationPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/location-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VirtualSensorsPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/virtual-sensors-page.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RecordScreenPage</class>
   <extends>QWidget</extends>
   <header>android/skin/qt/extended-pages/record-screen-page.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cellularButton</tabstop>
  <tabstop>batteryButton</tabstop>
  <tabstop>telephoneButton</tabstop>
  <tabstop>dpadButton</tabstop>
  <tabstop>microphoneButton</tabstop>
  <tabstop>fingerButton</tabstop>
  <tabstop>virtSensorsButton</tabstop>
  <tabstop>spacer2Button</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>spacer1Button</tabstop>
  <tabstop>settingsButton</tabstop>
  <tabstop>recordScreenButton</tabstop>
  <tabstop>locationButton</tabstop>
  <tabstop>cell_delayBox</tabstop>
  <tabstop>set_keyboardBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
